[0m16:59:10.330227 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe1d7b6a980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe1d5b4ba30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe1d5b4bac0>]}


============================== 16:59:10.336132 | ea7fd9bf-ff1b-4a3d-8786-bf38ee91db9a ==============================
[0m16:59:10.336132 [info ] [MainThread]: Running with dbt=1.6.1
[0m16:59:10.337291 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/home/codespace/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:59:10.338335 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/home/codespace/.dbt')}
[0m16:59:10.339257 [info ] [MainThread]: Creating dbt configuration folder at 
[0m17:00:08.097913 [debug] [MainThread]: Starter project path: /usr/local/python/3.10.13/lib/python3.10/site-packages/dbt/include/starter_project
[0m17:00:08.113247 [info ] [MainThread]: 
Your new dbt project "nyc_parking_violations" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m17:00:08.114189 [info ] [MainThread]: Setting up your profile.
[0m17:00:13.529658 [info ] [MainThread]: No sample profile found for duckdb.
[0m17:00:13.530869 [debug] [MainThread]: Command `dbt init` succeeded at 17:00:13.530686 after 63.21 seconds
[0m17:00:13.531468 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe1d7b6a980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe1d9dc11b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe1d5b4bb80>]}
[0m17:00:13.532095 [debug] [MainThread]: Flushing usage events
[0m04:48:30.626200 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efe2535e980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efe2333bcd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efe2333bc10>]}


============================== 04:48:30.635220 | 4d9a500a-0255-49eb-8c87-ce82ed4d3067 ==============================
[0m04:48:30.635220 [info ] [MainThread]: Running with dbt=1.6.1
[0m04:48:30.636134 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/codespace/.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m04:48:30.636885 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/DE_dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m04:48:30.637771 [debug] [MainThread]: Command `dbt run` failed at 04:48:30.637647 after 0.02 seconds
[0m04:48:30.638327 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efe2535e980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efe2333bc10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efe2333bd90>]}
[0m04:48:30.638898 [debug] [MainThread]: Flushing usage events
